queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+select * from animals where name like '%mon';
+select name from animals where date_of_birth between '2016-1-1' and '2019-1-1';
```

Comment:
- [ ] Kindly follow the requirement for the query `List the name of all animals born between 2016 and 2019."`.  Your query will find the animals whose born between 2016 and the first day of 2019.
 ### Hint:
```
Update the end date to '2019-12-31' the end day of 2019
```